app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -106,87 +125,45 @@ Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a
 
 You may be able to train part-time, but your part-time salary must be at least $nonuk_skilledworkervisatraineeminparttime$.
 
-### Your partner and children 
-
-If your course starts after 1 January 2024, you will not be able to bring your partner and children with you on a Student visa. 
-
-You may be able to bring your partner and children with you on a Skilled Worker visa. 
-
-### How a bursary or scholarship can help with your Student visa application 
-
-If you're training to teach physics or languages, you may be [eligible for a bursary or scholarship](/non-uk-teachers/fees-and-funding-for-non-uk-trainees?#bursaries-and-scholarships-for-non-uk-trainee-languages-and-physics-teachers). You can use them in your Student visa application as proof that you have enough money to support yourself. 
-
-## Check your status
-
-If you're not sure about your immigration status, you can [check if you need a UK visa](https://www.gov.uk/check-uk-visa).
-
-You will not need a visa to train to teach in the UK if you:
-
-* are an Irish citizen
-* have settled or pre-settled status under the [EU Settlement Scheme (EUSS)](https://www.gov.uk/settled-status-eu-citizens-families)
-* have [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk) 
-
-You will not need to be sponsored for a Student or Skilled Worker visa if you are on a: 
-
-* [Afghan Citizens Resettlement or Afghan Relocations and Assistance Policy scheme](https://www.gov.uk/government/publications/afghanistan-resettlement-and-immigration-policy-statement/afghanistan-resettlement-and-immigration-policy-statement-accessible-version)
-* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa)  
-* [Family visa](https://www.gov.uk/uk-family-visa)  
-* [Graduate visa](https://www.gov.uk/graduate-visa)  
-* [High Potential Individual visa](https://www.gov.uk/high-potential-individual-visa)  
-* [Homes for Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)  
-* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa)  
-* [UK Ancestry visa](https://www.gov.uk/ancestry-visa)  
-* [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)  
-* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)  
-* [Youth Mobility Scheme visa](https://www.gov.uk/youth-mobility)  
 
+## What you need to do to apply for your visa 
+You can apply for your visa once you have a confirmed offer of a training place. 
+[Search for postgraduate teacher training courses](https://find-teacher-training-courses.service.gov.uk/) that offer visa sponsorship. 
 
+Email your chosen provider to check if they can sponsor you, then [apply for your teacher training course](/how-to-apply-for-teacher-training). 
```

Comment:
Do they need to wait for confirmation that the provider can sponsor them before they apply?

(NB I know this is prod already - can follow this up separately if easier)

Reply:
Resolving this - will be followed up separately